src/client/.babelrc

```diff
@@ -4,6 +4,7 @@
   ],
   "plugins": [
   	"@babel/plugin-proposal-object-rest-spread",
-  	"@babel/plugin-proposal-class-properties"
+  	"@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-optional-chaining"
```

Comment:
indentation is funny here, can you correct?

Reply:
weird... the formatter changed preset this time. I hope it's ok ?